extensions/docs/msal-node-extensions.md

```diff
@@ -63,35 +133,52 @@ const linuxPersistence = LibSecretPersistence.create(cachePath, serviceName, acc
 An unencrypted file persistence, which works across all platforms, is provided for convenience, although not recommended.
 
 ```js
-import { FilePersistence } from "@azure/msal-node-extensions";
+const { FilePersistence } = require("@azure/msal-node-extensions");
+
+const filePath = "path/to/cache/file.json";
+const filePersistence = await FilePersistence.create(filePath, loggerOptions);
+// Pass the persistence to msal config's cachePlugin
+msalConfig.cache.cachePlugin = new PersistenceCachePlugin(filePersistence);
+ // Initialize the electron authenticator
+authProvider = new AuthProvider(msalConfig); 
```

Comment:
```suggestion
const pca = new PublicClientApplication({
    auth: {
            clientId: "CLIENT_ID_HERE",
        },
    cache: {
            cachePlugin: new PersistenceCachePlugin(linuxPersistence);
        },
  });
```

Reply:
The reader doesn't know about our custom class `AuthProvider`, we should just instantiate `PublicClientApplication`